lang/cs/BUILD.md

```diff
@@ -77,29 +83,3 @@ modify `test_script` section as follows:
 
 `nuget restore` is necessary to install `xunit.runner.console` package, which by default is installed in `TestRunner.proj`.
 
-Instructions on Building and Testing REEF .NET from Scratch
```

Comment:
To be clear the instructions above that i removed were redundant and confusing, the goal is to have 1 set of instructions to get the csharp operational so yes it'd be great to test drive the new instructions